cmd/soroban-rpc/internal/methods/send_transaction.go

```diff
@@ -73,7 +73,7 @@ func NewSendTransactionHandler(
 		}
 		txHash := hex.EncodeToString(hash[:])
 
-		ledgerInfo, err := reader.GetLedgerRange(ctx)
+		ledgerInfo, err := ledgerReader.GetLedgerRange(ctx)
```

Comment:
I know this is not directly related to this PR, but here we can get the latestledger instead (which is currently cheaper). Since we only need the latest ledger.

Optionally, you can fix it later, as part of https://github.com/stellar/soroban-rpc/issues/233

Reply:
@2opremio Yeah I agree we dont need the range here. I will fix it as part of #233